content/tutorials/quickstart/transfer-nftokens.md

```diff
@@ -175,34 +169,59 @@ Connect to the ledger and get the wallet accounts.
 
 
 ```
-      async function createSellOffer() {
-        const standby_wallet = xrpl.Wallet.fromSeed(standbySeedField.value)
-        const operational_wallet = xrpl.Wallet.fromSeed(operationalSeedField.value)
-          let net = getNet()
-          const client = new xrpl.Client(net)
-          results = 'Connecting to ' + getNet() + '...'
-          document.getElementById('standbyResultField').value = results
-        await client.connect()
-          results += '\nConnected. Creating sell offer...'
-          document.getElementById('standbyResultField').value = results
+async function createSellOffer() {
+  const standby_wallet = xrpl.Wallet.fromSeed(standbySeedField.value)
+  const operational_wallet = xrpl.Wallet.fromSeed(operationalSeedField.value)
+  let net = getNet()
+  const client = new xrpl.Client(net)
+  results = 'Connecting to ' + getNet() + '...'
+  document.getElementById('standbyResultField').value = results
+  await client.connect()
+  results += '\nConnected. Creating sell offer...'
+  document.getElementById('standbyResultField').value = results
+```
 
+Compute the Expiration Date, if present. The expiration date represents the number of seconds after the Ripple Epoch that the offer should expire. Subtract the Ripple Epoch from the current system date divided by 1,000, and add the number of days (or fractions of a day) multiplied by 86400 to get the correct expiration date.
```

Comment:
This conversion is confusing (I didn't know that doing a division operation on a `Date()` object in JS implicitly converts it to milliseconds first) and a lot of this math could be simplified by using functions that are built into xrpl.js (see my comment on the actual code below).